js/src/bqplot.less

```diff
@@ -125,8 +125,21 @@
     .common();
     svg.bqplot {
         .axis {
-            rect, path, line {
-                stroke: black;
+            path, line {
+                stroke: #EEEEEE;
+            }
+            rect {
+                stroke: white;
+                opacity: 1.0;
+            }
+            .tick line {
+            	stroke: white;
+            	stroke-width: 1.4;
+                opacity: 1.0;
+            }
+            .tick text {
+                fill: #7f7f7f;
```

Comment:
If you remove that `fill` attribute, you retrieve the dark tick marks like in ggplot.